pages/CardDirectoryPage.tsx

```diff
@@ -1,18 +1,18 @@
 import { Box, Typography, useMediaQuery, useTheme } from '@mui/material';
+import type { Theme } from '@mui/system';
+import type { ReactElement } from 'react';
```

Comment:
**thought | nitpick:** We should establish a standard order for imports/types

Reply:
Its kinda weird it seems like the linter is suddenly kicking in and organizing them all. Not sure why. Ill open a separate PR for this and see whats happeneing.